data-dashboard/server/services/chartSerializer.ts

```diff
@@ -0,0 +1,33 @@
+import xlsx from 'xlsx';
+import { Chart } from '../models';
+import { StorageService } from './storageService';
+
+export default class ChartSerializer {
+  private readonly storageService: StorageService;
+
+  constructor(storageService: StorageService) {
+    this.storageService = storageService;
+  }
+
+  async buildChart(fileHandle: string): Promise<Chart> {
+    const file = await this.storageService.readFile(fileHandle);
+    const workbook = xlsx.read(file, { type: 'buffer' });
+    for (const sheet of workbook.SheetNames) {
+      const worksheet = workbook.Sheets[sheet];
+      const asJson = xlsx.utils.sheet_to_json(worksheet, { header: 1 });
+      console.log({ asJson });
+    }
+
+    return Promise.resolve({
+      boardId: '1',
```

Comment:
I don't like hardcoded id's because it could crash, I think we could remove this for this PR and complete it in the next one

Reply:
I removed any references to the chartSerializer service, so it is not exposed and it does not cause any bugs. What do you think?

Reply:
nicee